src/Common/src/System/Security/Cryptography/Asn1/GeneralNameAsn.cs

```diff
@@ -70,7 +70,7 @@ internal struct OtherNameAsn
         [ObjectIdentifier]
         internal string TypeId;
 
-        [ExpectedTag(0, ExplicitTag = true)]
+        [ExpectedTag(0)]
```

Comment:
The RFC says that this is explicit, so why remove it here?  It then looks like the code working with it later explicitly puts it back in.

Reply:
Hmm, will look into it. It was failing the unit tests with the ExplicitTag = true, so there are some test data for it.

Reply:
You were right. I reverted this change and removed the code that was explicitly adding it and it still passes all the tests.